common/app/shoppingcart/AddressBean.java

```diff
@@ -28,54 +25,26 @@ public AddressBean() {
 
     public AddressBean(@Nullable final Address address, final Locale locale) {
         if (address != null) {
-            try {
-                BeanUtils.copyProperties(this, address);
-            } catch (IllegalAccessException | InvocationTargetException e) {
-                throw new RuntimeException(e);
-            }
-            setCountryCode(address.getCountry().getAlpha2());
-            setCountry(address.getCountry().toLocale().getDisplayCountry(locale));
+            this.title = address.getTitle();
```

Comment:
I like to use `this` always for constructors, it allows me to see it clearly when the assignment happened. Just a little hint for me.